cmd/net/src/lib.rs

```diff
@@ -57,7 +57,17 @@ enum NetCommand {
     /// Print the link status
     Status,
     /// Print the counters
-    Counters,
+    ///
+    /// This is a destructive operation that clears counter values, resetting
+    /// them to zero.
+    Counters {
+        /// Show a table of raw counter values
+        #[clap(short, long)]
+        table: bool,
+        /// Show a diagram of good packet counts in context
```

Comment:
very nitpicky: do you think the help text should state that both are displayed if neither flags are present?

Reply:
Sure, done in e89180467001c17d22745957982a6fb3068f0fb4

(also, TIL that `-h` and `--help` are different in `clap`)